content/applications/services/helpdesk/advanced/track_and_bill.rst

```diff
@@ -0,0 +1,244 @@
+===================
+Track and bill time
+===================
+
+Odoo *Helpdesk* provides teams with the ability to track the amount of hours spent working on a
+ticket, and to bill a customer for that time. Through integrations with the *Sales*, *Timesheets*
+and *Accounting* applications, customers can be charged once the work is completed, or before it
+has even begun.
+
+.. warning::
+   Since the :guilabel:`Track & Bill Time` features require integration with other applications,
+   enabling them may result in the installation of additional modules (or applications).
+
+   Installing a new application on a *One-App-Free* database triggers a 15-day trial. At the
+   end of the trial, if a paid subscription has not been added to the database, it will no longer
+   be active or accessible.
+
+Configure track and bill time features
+======================================
+
+Before a customer can be invoiced for support services, the *Track & Bill Time* features must first
+be enabled. These features must be enabled on each *Helpdesk* team where they will be utilized.
+
+Enable track and bill time on a helpdesk team
+---------------------------------------------
+
+To view and enable the :guilabel:`Track & Bill Time` features on a *Helpdesk* team, first navigate
+to :menuselection:`Helpdesk --> Configuration --> Teams`. Then select a team from the list or
+:doc:`create a new one </applications/services/helpdesk/overview/getting_started>`. This will
+reveal a team's settings page.
+
+On the team's settings page, scroll to the :guilabel:`Track & Bill Time` section. Check the boxes
+labeled :guilabel:`Timesheets` and :guilabel:`Time Billing`.
+
+Checking the box for :guilabel:`Timesheets` causes a new field to appear, labeled *Project*.
```

Comment:
we will apply a :guilabel: to Project here because it's appearance is part of the active instruction